safe/impact_functions/generic/classified_polygon_people/metadata_definitions.py

```diff
@@ -0,0 +1,104 @@
+from safe.definitions import (
+    layer_mode_classified,
+    layer_geometry_polygon,
+    hazard_all,
+    hazard_category_single_event,
+    exposure_area,
+    generic_vector_hazard_classes,
+)
+from safe.utilities.i18n import tr
+from safe.impact_functions.generic.\
+    classified_polygon_people import parameter_definitions
+
+
+class ClassifiedPolygonHazardPolygonPeopleFunctionMetadata(ImpactFunctionMetadata):
+
+    @staticmethod
+    def as_dict():
+        """Return metadata as a dictionary.
+
+        This is a static method. You can use it to get the metadata in
+        dictionary format for an impact function.
+
+        :returns: A dictionary representing all the metadata for the
+            concrete impact function.
+        :rtype: dict
+        """
+        dict_meta = {
+            'id': 'ClassifiedPolygonHazardPolygonPeopleFunction',
+            'name': tr('Classified polygon hazard on polygon people'),
+            'impact': tr('Be affected'),
+            'title': tr('Be affected'),
+            'function_type': 'qgis2.0',
+            'author': 'Samweli Twesa Mwakisambwe(smwltwesa6@gmail.com)',
+            'date_implemented': '06/08/2015',
+            'overview': tr(
+                'To assess the impact of each hazard zone on polygon people.'),
+            'detailed_description': '',
+            'hazard_input': tr(
+                'The hazard layer must be a polygon layer. This layer '
+                'must have an attribute representing the hazard '
+                'zone that can be specified in the impact function options.'),
+            'exposure_input': tr(
+                'Vector polygon layer where each '
+                'polygon represents a type of area where people lives.'),
+            'output': tr(
+                'A vector layer of areas polygons with each tagged '
+                'according to the hazard zone in which it falls.'),
+            'actions': tr(
+                'Provide details about how big area fall within '
+                'each hazard zone.'),
+            'limitations': [],
+            'citations': [],
+            'layer_requirements': {
+                'hazard': {
+                    'layer_mode': layer_mode_classified,
+                    'layer_geometries': [layer_geometry_polygon],
+                    'hazard_categories': [
+                        hazard_category_single_event
+                    ],
+                    'hazard_types': hazard_all,
+                    'continuous_hazard_units': [],
+                    'vector_hazard_classifications': [
+                        generic_vector_hazard_classes],
+                    'raster_hazard_classifications': [],
+                    'additional_keywords': []
+                },
+                'exposure': {
+                    'layer_mode': layer_mode_classified,
+                    'layer_geometries': [layer_geometry_polygon],
+                    'exposure_types': [exposure_area],
```

Comment:
We should be referencing the people column in the attribute table here shouldn't we?